src/components/ButtonPanel.js

```diff
@@ -3,13 +3,39 @@ import React from 'react';
 import Button from './Button';
 
 
-class ButtonPanel extends React.Component {
-  render() {
-    return (
-      <Button onClick={this.onClick} />
-    );
-  }
-}
+const ButtonPanel = () => (
+  <div className="calculator_button">
+    <div className="groupOne">
+      <Button aria-label="AC" name="AC" className="tierOne" />
+      <Button aria-label="+/-" name="+/-" className="tierOne" />
+      <Button aria-label="%" name="%" className="tierOne" />
+      <Button aria-label="/" name="÷" className="key--operator" id="divide" />
+    </div>
+    <div className="groupTwo">
+      <Button aria-label="7" name="7" className="tierOne" />
+      <Button aria-label="8" name="8" className="tierOne" />
+      <Button aria-label="9" name="9" className="tierOne" />
+      <Button aria-label="x" name="x" className="key--operator" id="multiply" />
+    </div>
+    <div className="groupThree">
+      <Button aria-label="4" name="4" className="tierOne" />
```

Comment:
[Optional]

We could rewrite the ButtonPanel component to use two loops to make the `Button` instead of hardcoding them. By doing this, we keep our code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

We could achieve this by creating an array of symbol and loop through the array using [`map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.